usr/sbin/p11sak/p11sak.c

```diff
@@ -2212,12 +2233,13 @@ static CK_RV build_opts(const struct p11sak_opt *opts,
 static CK_RV build_cmd_opts(const struct p11sak_opt *cmd_opts,
                             char **optstring, struct option **longopts)
 {
-    unsigned int optstring_len = 0, longopts_count = 0;
     CK_RV rc;
-
+    unsigned int optstring_len = 0;
```

Comment:
why these changes ? It results in the same that it was before ....

Reply:
Originally removed because `getopt_long` was missing. I'd bunched the `count_opts` call with the definition so I could hide them both behind a single `#ifdef`.

Reply:
Everywhere (not just this file) you see lines rearranged that relate to `getopt_long`, this is the reason why.

Reply:
OK, I would prefer that you revert these changes then, now that you do have longopt support.